### PR TITLE
Builder Refinement (story2018)

### PR DESCRIPTION
**Pull request by Olaniyan99**
builder1
builder2

### CODE REVIEW

File: src/app/onion/learning-object-builder/components/standard-outcomes/standard-outcomes.component.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <div class="standard-outcomes">
-  <div class="title">Standard Outcomes</div>
+  <div class="title">Curricular Guidelines</div>
   <div class="search-bar">
-    <input type="search" placeholder="Search standard outcomes..." [(ngModel)]="searchStringValue" (ngModelChange)="searchString$.next($event)" id="search" aria-label="Search for standard outcomes"/>
+    <input type="search" placeholder="Search curricular guidelines..." [(ngModel)]="searchStringValue" (ngModelChange)="searchString$.next($event)" id="search" aria-label="Search for standard outcomes"/>
```
Comment (pzalep1) [inline src/app/onion/learning-object-builder/components/standard-outcomes/standard-outcomes.component.html:4]:
```suggestion
    <input type="search" placeholder="Search curricular guidelines..." [(ngModel)]="searchStringValue" (ngModelChange)="searchString$.next($event)" id="search" aria-label="Search for curricular guidelines"/>
```

File: src/app/onion/learning-object-builder/pages/outcome-page/outcome-page.component.html [hunk 2]
```diff
@@ -20,6 +20,9 @@
         (activate)="setActiveOutcome(o.id)"
       >
       </clark-outcome>
+      <div class="addButton" >
```
Comment (pzalep1) [inline src/app/onion/learning-object-builder/pages/outcome-page/outcome-page.component.html:23]: What is the reasoning behind the wrapper for the button?

Comment (Olaniyan99) [reply src/app/onion/learning-object-builder/pages/outcome-page/outcome-page.component.html:23]: we spoke about not have the button just sit there. we it to have a white background to complete the card. so decided to style the div.

**Comment by Olaniyan99** [comment]
Zero_Outcome
Single_Outcome
Multiple_Outcome

File: src/app/onion/learning-object-builder/pages/outcome-page/outcome-page.component.html [hunk 3]
```diff
@@ -2,10 +2,9 @@
   <div left>
     <clark-scaffold *ngIf="learningObject" [learningObject]= "learningObject"></clark-scaffold>
   </div>
-  <div main>
+  <div main class="main">
     <div class="top">
-      <div class="title">Learning Outcomes</div>
-      <button id="form" class="button good" aria-label="Add New Outcome" (activate)="newOutcome()">New <i class="fas fa-plus"></i></button>
+      <div class="title" *ngIf = "iterableOutcomes.length !== 1; else singleOutcome">{{iterableOutcomes.length}} Learning Outcomes</div>
```
Comment (pzalep1) [inline src/app/onion/learning-object-builder/pages/outcome-page/outcome-page.component.html:7]: You can also conditionally add an s in the span and you don't need to use the template

File: src/app/onion/learning-object-builder/pages/outcome-page/outcome-page.component.ts [hunk 4]
```diff
@@ -143,6 +149,19 @@ export class OutcomePageComponent implements OnInit, OnDestroy {
     );
   }
 
+  validateNewOutcome() {
```
Comment (pzalep1) [inline src/app/onion/learning-object-builder/pages/outcome-page/outcome-page.component.ts:152]: Can we add a comment here?

File: src/app/onion/learning-object-builder/pages/outcome-page/outcome-page.component.html [hunk 5]
```diff
@@ -2,10 +2,9 @@
   <div left>
     <clark-scaffold *ngIf="learningObject" [learningObject]= "learningObject"></clark-scaffold>
   </div>
-  <div main>
+  <div main class="main">
     <div class="top">
-      <div class="title">Learning Outcomes</div>
-      <button id="form" class="button good" aria-label="Add New Outcome" (activate)="newOutcome()">New <i class="fas fa-plus"></i></button>
+      <div class="title">{{iterableOutcomes.length}} Learning Outcome<span *ngIf = "iterableOutcomes.length !== 1">s</span></div>
```
Comment (pzalep1) [inline src/app/onion/learning-object-builder/pages/outcome-page/outcome-page.component.html:7]:
```suggestion
      <div class="title">{{iterableOutcomes?.length}} Learning Outcome<span *ngIf = "iterableOutcomes?.length !== 1">s</span></div>
```

File: src/app/onion/learning-object-builder/pages/outcome-page/outcome-page.component.ts [hunk 6]
```diff
@@ -143,6 +149,20 @@ export class OutcomePageComponent implements OnInit, OnDestroy {
     );
   }
 
+  // This functions validates that an outcome is valid. if valid, enables the Add Outcome buton
```
Comment (pzalep1) [inline src/app/onion/learning-object-builder/pages/outcome-page/outcome-page.component.ts:152]:
```suggestion
  // This functions validates that an outcome is valid. If valid, enables the Add Outcome button
```

Comment (Olaniyan99) [reply src/app/onion/learning-object-builder/pages/outcome-page/outcome-page.component.ts:152]: Thanks

**Pull request merged**